glitter/publisher/utils.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.utils import timezone
+
+from .models import PublishAction
+
+
+def process_actions(action_ids=None):
+    """
+    Process actions in the publishing schedule.
+
+    Returns the number of actions processed.
+    """
+    actions_taken = 0
+    action_list = PublishAction.objects.prefetch_related(
```

Comment:
This seems to work fine for me.

You'll need to add the inline onto 2 different models, then this statement does something similar to following:

- Select all publishactions
- Select all pages for publishactions where content type is page
- Select all news posts for publishactions where content type is news post
- etc.